share/completions/solc.fish

```diff
@@ -0,0 +1,83 @@
+# General Information
+complete -c solc -l help -d "Show help message and exit."
+complete -c solc -l version -d "Show version and exit."
+complete -c solc -l license -d "Show licensing information and exit."
+complete -c solc -l input-file -d "input file"
+
+# Input Options
+complete -c solc -l base-path -d "Use the given path as the root of the source tree instead of the root of the filesystem."
+complete -c solc -l include-path -d "Make an additional source directory available to the default import callback. Use this option if you want to import contracts whose location is not fixed in relation to your main source tree, e.g. third-party libraries installed using a package manager. Can be used multiple times. Can only be used if base path has a non-empty value."
```

Comment:
this is quite long - fish will only show the first 60 or so characters at most.
The descriptions are not reference documentation (like man pages) but only
reminders

```suggestion
complete -c solc -l include-path -d "Add extra directory to import conctracts from"
```

Please also check other descriptions. Short is good here